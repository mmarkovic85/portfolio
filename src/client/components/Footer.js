import React from "react";

const Footer = () => (
  <footer>
    <h6>Footer placeholder</h6>
  </footer>
);

export {
  Footer as default
};
