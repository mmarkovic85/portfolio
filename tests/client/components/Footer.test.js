import React from "react";
import renderer from "react-test-renderer";
import Footer from "../../../src/client/components/Footer";

test("Should render Footer component correctly", () => {
  const rendered = renderer.create(
    <Footer />
  );

  expect(rendered.toJSON()).toMatchSnapshot();
});
