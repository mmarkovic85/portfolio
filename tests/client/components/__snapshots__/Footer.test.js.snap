// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Footer component correctly 1`] = `
<footer>
  <h6>
    Footer placeholder
  </h6>
</footer>
`;
